app/uk/gov/hmrc/plasticpackagingtax/registration/connectors/DeregistrationConnector.scala

```diff
@@ -44,7 +44,7 @@ class DeregistrationConnector @Inject() (
     ).andThen { case _ => timer.stop() }
       .map {
         case response @ HttpResponse(OK, _, _) =>
-          Right()
+          Right(())
```

Comment:
its technically correct, proably to stop a compile warning, is because we have an Either[_, Unit]. `Right()` is the compiling assuming we mean `Right(())`.
Does pose a good point of where we actually need an either here as we dont care about the response, beyond the scope of this PR thou for sure :)

Reply:
Yes this just to stop the warning as automatic unit inference has been deprecated since scala 2.11. Just trying to geet read of few warning as there also a lot of unused import.